tests/acceptance/features/apiActivities/shareActivities.feature

```diff
@@ -1076,3 +1076,48 @@ Feature: check share activity
         }
       }
       """
+
+  @issue-9849
+  Scenario: sharee tries to check activities of folder using share mountpoint id
+    Given user "Alice" has created folder "/FOLDER"
+    And user "Alice" has sent the following resource share invitation:
+      | resource        | /FOLDER  |
+      | space           | Personal |
+      | sharee          | Brian    |
+      | shareType       | user     |
+      | permissionsRole | Editor   |
+    And user "Brian" has a share "/FOLDER" synced
+    And user "Brian" has uploaded file with content "some data" to "Shares/FOLDER/newfile.txt"
+    And user "Brian" has uploaded file with content "edited data" to "Shares/FOLDER/newfile.txt"
+    And user "Brian" has deleted file "Shares/FOLDER/newfile.txt"
+    When user "Brian" lists the activities for folder "FOLDER" of space "Shares" using the Graph API
```

Comment:
could you update this step?
```suggestion
    When user "Brian" lists the activities of folder "FOLDER" from space "Shares" using the Graph API
```

Reply:
@saw-jan 
May be we should update this step as well?
``` feature
When user "Alice" lists the activities for folder "New Folder" of space "Personal" with depth "-1" using the Graph API
```

Reply:
yes, please